dsc_lib/src/functions/reference.rs

```diff
@@ -12,6 +12,14 @@ use tracing::debug;
 pub struct Reference {}
 
 impl Function for Reference {
+    fn description(&self) -> String {
+        t!("functions.reference.description").to_string()
+    }
+
+    fn category(&self) -> FunctionCategory {
+        FunctionCategory::Deployment
```

Comment:
https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions#resource-functions lists reference under the Resource category